Dockerfile

```diff
@@ -27,9 +28,9 @@ RUN DEBIAN_FRONTEND=noninteractive dpkg --add-architecture i386 \
 
 # Install the Android SDK
 RUN cd /opt && \
-    wget -q https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz -O android-sdk.tgz && \
-    tar -xvzf android-sdk.tgz && \
-    rm -f android-sdk.tgz
+    wget -q https://dl.google.com/android/repository/tools_r25.1.7-linux.zip -O android-tools-linux.zip && \
+    unzip android-tools-linux.zip -d ${ANDROID_HOME} && \
```

Comment:
Are you sure this use of environment variables is allowed? The reference for ENV does not mention RUN as one of the commands that can use such variables: https://docs.docker.com/engine/reference/builder/